src/promptflow-core/promptflow/core/_flow.py

```diff
@@ -377,6 +376,7 @@ def __call__(self, *args, **kwargs):
         """
         if args:
             raise UserErrorException("Prompty can only be called with keyword arguments.")
+        enrich_prompt_template(self._template, variables=kwargs)
```

Comment:
Should we only filter the variables used in the template?

Reply:
The kwargs passed in here are all used for template render